YoCode/Directory.cs

```diff
@@ -1,20 +1,40 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.IO;
 
 namespace YoCode
 {
-
-    
-    public class Directory
+    public class Directory : IDirectory
     {
-        public List<String> OriginalPaths { get; } = null;
-        public List<String> ModifiedPaths { get; } = null;
+        public List<String> OriginalPaths { get; }
+        public List<String> ModifiedPaths { get; }
 
         public Directory(List<String> originalPaths, List<String> modifiedPaths)
         {
             OriginalPaths = originalPaths;
             ModifiedPaths = modifiedPaths;
         }
 
+        private List<Stream> ReturnPathFileStream(List<string> paths)
+        {
+            List<Stream> streamList = new List<Stream>();
+
+            for (int i = 0; i < paths.Count; i++)
+            {
+                FileStream fs = File.Create(paths[i]);
+                streamList.Add(fs);
+            }
```

Comment:
This `for` loop should probably be
```
foreach(var path in paths)
{
    streamList.Add(File.OpenWrite(path));
}
```
`File.Create` will overwrite a file if it exists and you don't want to do that, you want to open it.